lib/Common/Memory/RecyclerPointers.h

```diff
@@ -515,13 +520,23 @@ void qsort_s(T* arr, size_t count, const Comparer& comparer, void* context)
     typedef typename AllocatorWriteBarrierPolicy<Allocator, ItemPolicy>::Policy Policy;
     _QuickSortImpl<Policy>::Sort(arr, count, comparer, context);
 }
+
+#ifndef _WIN32
+template<class T, class PolicyType = T, class Allocator = Recycler, class Comparer>
+void qsort_s(T* arr, size_t count, size_t size, const Comparer& comparer, void* context)
+{
+    typedef typename _ArrayItemWriteBarrierPolicy<PolicyType>::Policy ItemPolicy;
+    typedef typename AllocatorWriteBarrierPolicy<Allocator, ItemPolicy>::Policy Policy;
+    _QuickSortImpl<Policy>::Sort(arr, count, comparer, context, size);
+}
+#else
```

Comment:
Why add this? It is already typed and does not need "size". The existing version above already meets requirements, and with a different signature, it requires the user to double examine this call (make sure to use correct PolicyType -- default T might be incorrect). Please don't make this "xplat vs. Windows" difference.